examples/all-clusters-app/linux/main-common.cpp

```diff
@@ -210,7 +210,10 @@ void ApplicationShutdown()
 using namespace chip::app::Clusters::LaundryWasherControls;
 void emberAfLaundryWasherControlsClusterInitCallback(EndpointId endpoint)
 {
-    LaundryWasherControlsServer::SetDefaultDelegate(1, &LaundryWasherControlDelegate::getLaundryWasherControlDelegate());
+    LaundryWasherControlDelegate & laundryWasherControlDelegate = LaundryWasherControlDelegate::getLaundryWasherControlDelegate();
+
+    LaundryWasherControlsServer::SetDefaultDelegate(endpoint, &laundryWasherControlDelegate);
+    laundryWasherControlDelegate.Init(endpoint);
```

Comment:
Shouldn't we init it before registering it?

Reply:
Yes. Will fix it